pkgs/build-support/trivial-builders/default.nix

```diff
@@ -887,9 +887,8 @@ rec {
   /* An immutable file in the store with a length of 0 bytes. */
   emptyFile = runCommand "empty-file"
     {
-      outputHashAlgo = "sha256";
+      outputHash = "sha256-d6xi4mKdjkX2JFicDIv5niSzpyI0m/Hnm8GGAIU04kY=";
       outputHashMode = "recursive";
-      outputHash = "0ip26j2h11n1kgkz36rl4akv694yz65hr72q4kv4b3lxcbi65b3p";
       preferLocalBuild = true;
     } "touch $out";
```

Comment:
Digression:

I've learned about `builtins.path`, which as I understand it would be an alternate implementation that didn't depend on the `runCommand` machinery.